Makefile

```diff
@@ -247,11 +247,9 @@ install: milvus
 	@echo "Installing binary to './bin'"
 	@mkdir -p $(GOPATH)/bin && cp -f $(PWD)/bin/milvus $(GOPATH)/bin/milvus
 	@mkdir -p $(LIBRARY_PATH)
-ifeq ($(OS),Darwin)
-	@cp -r -P $(PWD)/internal/core/output/lib/*.dylib* $(LIBRARY_PATH)
-else
-	@cp -r -P $(PWD)/internal/core/output/lib/*.so* $(LIBRARY_PATH)
-endif
+	-cp -r -P $(PWD)/internal/core/output/lib/*.dylib* $(LIBRARY_PATH)
+	-cp -r -P $(PWD)/internal/core/output/lib/*.so* $(LIBRARY_PATH)
+	-cp -r -P $(PWD)/internal/core/output/lib64/*.so* $(LIBRARY_PATH)
```

Comment:
```
Building Milvus ...
Installing binary to './bin'
cp -r -P /home/yangxuan/Github/milvus/internal/core/output/lib/*.dylib* /home/yangxuan/Github/milvus/lib
cp: 无法获取'/home/yangxuan/Github/milvus/internal/core/output/lib/*.dylib*' 的文件状态(stat): 没有那个文件或目录
make: [Makefile:250：install] 错误 1 (已忽略）
cp -r -P /home/yangxuan/Github/milvus/internal/core/output/lib/*.so* /home/yangxuan/Github/milvus/lib
cp -r -P /home/yangxuan/Github/milvus/internal/core/output/lib64/*.so* /home/yangxuan/Github/milvus/lib
cp: 无法获取'/home/yangxuan/Github/milvus/internal/core/output/lib64/*.so*' 的文件状态(stat): 没有那个文件或目录
make: [Makefile:252：install] 错误 1 (已忽略）
Installation successful.

```

Reply:
This looks terrible at the end of `make install`, even though the result is correct.

Reply:
It's hard to judge OS_NAME in Makefile, so I use the hack way 😏

Reply:
A suggestion:
```
-cp -r -P ... 2>/dev/null 
```

Reply:
slightly better
```
cp -r -P /home/yangxuan/Github/milvus/internal/core/output/lib/*.dylib* /home/yangxuan/Github/milvus/lib 2>/dev/null
make: [Makefile:250：install] 错误 1 (已忽略）
cp -r -P /home/yangxuan/Github/milvus/internal/core/output/lib/*.so* /home/yangxuan/Github/milvus/lib 2>/dev/null
cp -r -P /home/yangxuan/Github/milvus/internal/core/output/lib64/*.so* /home/yangxuan/Github/milvus/lib 2>/dev/null
make: [Makefile:252：install] 错误 1 (已忽略）

```